src/polyfills.ts

```diff
@@ -51,8 +53,7 @@ import 'core-js/es7/reflect';
  * Only required if AnimationBuilder is used within the application and using IE/Edge or Safari.
  * Standard animation support in Angular DOES NOT require any polyfills (as of Angular 6.0).
  */
-// import 'web-animations-js';  // Run `npm install --save web-animations-js`.
-
+import 'web-animations-js';  // Run `npm install --save web-animations-js`.
```

Comment:
I don't think this is required as we don't use animations

Reply:
ok, will remove